### PR TITLE
Normative and non-normative documentation changes

### LINKED ISSUE #107: Check use of normative words
Like in the normative review for OCDS, please check whether normative keywords (must, should, recommended, etc.) are used consistently, and (to the extent possible) only in the normative sections of the documentation.

### LINKED ISSUE #351: Project identifiers: Update schema description to clarify normative statements, move prefix documentation to schema reference section
The description of `id` could be improved to make it clearer (through the use of normative keywords) that the identifier must be unique and must begin with a registered project identifier prefix.

Also, the [prefix documentation and list of registered prefixes](https://standard.open-contracting.org/infrastructure/latest/en/guidance/identifiers/#project-identifier-prefixes) should be moved to the reference section of the documentation, since the list of registered prefixes is normative.

Current description:

> A unique identifier or reference number for this infrastructure project, composed of a project identifier prefix and a local project identifier. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/0.9/en/guidance/identifiers/#project-identifier-prefixes).

Proposed description:

> An identifier for this infrastructure project. The identifier must be globally unique and must begin with a registered project identifier prefix. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/0.9/en/guidance/identifiers/#project-identifier-prefixes).

cc @neelima-j

### PR DESCRIPTION
**Pull request by neelima-j**
Closes https://github.com/open-contracting/infrastructure/issues/107. Closes https://github.com/open-contracting/infrastructure/issues/351. Related discussion in https://crm.open-contracting.org/issues/7853.

Made changes to documentation in line with norms for normative and non-normative words ([review spreadsheet](https://docs.google.com/spreadsheets/d/101MPk_SWgQaK_vYhvkoa1P9TeOHnoa3nDzuZ0WTu1Lw/edit#gid=0))/

### CODE REVIEW

**Review by jpmckinney — COMMENTED** [review]
Looks good! I haven't compared to the spreadsheet, but I assume all the changes were made and are identical.

Please update `docs/reference/changelog.md` with a message similar to those in https://standard.open-contracting.org/latest/en/history/changelog/

**Comment by duncandewhurst** [comment]
I added two commits to close https://github.com/open-contracting/infrastructure/issues/351

File: schema/project-level/project-schema.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "properties": {
     "id": {
       "title": "Identifier or Reference",
-      "description": "A unique identifier or reference number for this infrastructure project, composed of a project identifier prefix and a local project identifier. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/guidance/identifiers/#project-identifier-prefixes).",
+      "description": "An identifier for this infrastructure project. The identifier must be globally unique and must begin with a registered project identifier prefix. For more information, see the [project identifier guidance](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/prefixes).",
```
Comment (jpmckinney) [inline schema/project-level/project-schema.json]: Hmm, this link is not "project identifier guidance", it's a list of prefixes. The guidance is still in the guidance section.

Comment (duncandewhurst) [reply schema/project-level/project-schema.json]: Ah, yes. Good spot. Fixed in https://github.com/open-contracting/infrastructure/pull/355/commits/f1686d2efb0c77f13af0060f308efebf540e7e66

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**